### PR TITLE
feat: 🎸 add investorCount method in TransferRestriction Count

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Add investorCount method to Count namespace

### Breaking Changes

N/A

### JIRA Link

[DA-327](https://polymath.atlassian.net/browse/DA-327)

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/entities/Asset/TransferRestrictions/Count.ts [hunk 1]
```diff
@@ -53,7 +61,7 @@ export class Count extends TransferRestrictionBase<TransferRestrictionType.Count
    * Enables an investor count statistic for the Asset, which is required before creating restrictions
    *
    * The counter is only updated automatically with each transfer of tokens after the stat has been enabled.
-   * As such the initial value for the stat should be passed in
+   * As such the initial value for the stat should be passed in, which can be fetched with {@link Count.investorCount }
```
Comment (prashantasdeveloper) [inline src/api/entities/Asset/TransferRestrictions/Count.ts:64]: If this gets merged after alpha -> beta, @link tag would require some changes to support wiki linking

File: src/api/entities/Asset/TransferRestrictions/Count.ts [hunk 2]
```diff
@@ -80,4 +88,13 @@ export class Count extends TransferRestrictionBase<TransferRestrictionType.Count
    *   before reaching that limit
    */
   public declare get: () => Promise<ActiveTransferRestrictions<CountTransferRestriction>>;
+
+  /**
+   * Returns the count of individual holders of the Asset
+   *
+   * @note This value can be used to initialize enableStat. If used for this purpose there is a potential race condition
```
Comment (prashantasdeveloper) [inline src/api/entities/Asset/TransferRestrictions/Count.ts:95]:
```suggestion
   * @note This value can be used to initialize `enableStat`. If used for this purpose there is a potential race condition
```

**Review by prashantasdeveloper — COMMENTED** [review]
Left some nit comments, otherwise LGTM

File: src/api/entities/Asset/TransferRestrictions/Count.ts (hunk 1 above)
Comment (VictorVicente) [reply src/api/entities/Asset/TransferRestrictions/Count.ts:64]: @polymath-eric could you fix this as @prashantasdeveloper mentions?

**Pull request merged**